src/main/java/io/vlingo/actors/plugin/mailbox/MailboxProviderKeeper.java

```diff
@@ -0,0 +1,88 @@
+// Copyright © 2012-2018 Vaughn Vernon. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the
+// Mozilla Public License, v. 2.0. If a copy of the MPL
+// was not distributed with this file, You can obtain
+// one at https://mozilla.org/MPL/2.0/.
+
+package io.vlingo.actors.plugin.mailbox;
+
+import io.vlingo.actors.Mailbox;
+import io.vlingo.actors.MailboxProvider;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class MailboxProviderKeeper implements io.vlingo.actors.MailboxProviderKeeper  {
```

Comment:
I am a little concerned about the name clash that may cause confusion. Yes, I understand that these are in different packages. Hows about naming the class `DefaultMailboxProviderKeeper`?

Reply:
That's a good point, I'm going to change it.